### ISSUE TITLE
Errors when opening routing service sample

### ISSUE THREAD

**Issue by uthaiyashankar** [ballerina-platform/ballerina-lang#6314]
<a href="https://github.com/yasassri"><img src="https://avatars1.githubusercontent.com/u/7681361?v=4" align="left" width="96" height="96" hspace="10"></img></a> **Issue by [yasassri](https://github.com/yasassri)**
_Thursday Oct 12, 2017 at 10:07 GMT_
_Originally opened as https://github.com/ballerina-lang/composer/issues/3581_

----

Following error was observed.

```
Oct 12, 2017 3:34:45 PM org.ballerinalang.composer.service.workspace.rest.datamodel.BLangFileRestService generateJSON
SEVERE:  [json]
Oct 12, 2017 3:34:45 PM org.ballerinalang.composer.service.workspace.rest.datamodel.BLangFileRestService generateJSON
SEVERE:  [string, TypeCastError]
Oct 12, 2017 3:34:45 PM org.ballerinalang.composer.service.workspace.rest.datamodel.BLangFileRestService generateJSON
SEVERE:  [string, TypeCastError]
Oct 12, 2017 3:34:45 PM org.ballerinalang.composer.service.workspace.rest.datamodel.BLangFileRestService generateJSON
SEVERE:  [ballerina.net.http:Response]
Oct 12, 2017 3:34:45 PM org.ballerinalang.composer.service.workspace.rest.datamodel.BLangFileRestService generateJSON
SEVERE:  [ballerina.net.http:Response]
```

**Comment by uthaiyashankar** [comment]
<a href="https://github.com/yasassri"><img src="https://avatars1.githubusercontent.com/u/7681361?v=4" align="left" width="48" height="48" hspace="10"></img></a> **Comment by [yasassri](https://github.com/yasassri)**
_Thursday Oct 12, 2017 at 10:08 GMT_

----

Seems most of the samples are broken.

**Comment by uthaiyashankar** [comment]
<a href="https://github.com/dilinisg"><img src="https://avatars2.githubusercontent.com/u/1845370?v=4" align="left" width="48" height="48" hspace="10"></img></a> **Comment by [dilinisg](https://github.com/dilinisg)**
_Thursday Oct 12, 2017 at 10:16 GMT_

----

Same issue observed in windows
![2](https://user-images.githubusercontent.com/1845370/31491296-7b5f5d02-af64-11e7-8704-8d59c185fd83.PNG)

**Comment by hevayo** [comment]
Fixed in 0.970 we replace the samples with BBE.
